person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] `rental` should be marked with `attr_accessor`, as we will be both _accessing_ it as well as _modifying_ it. Kindly ensure to improve upon this aspect.